src/api/database/bulk-api.rst

```diff
@@ -456,6 +456,139 @@ Sending multiple queries to a database
     The multiple queries are also supported in /db/_local_docs/queries and
     /db/_design_docs/queries (similar to /db/_all_docs/queries).
 
+.. _api/db/bulk_get:
+
+==================
+``/db/_bulk_get``
+==================
+
+.. http:post:: /{db}/_bulk_get
+    :synopsis: Fetches several documents at the given revision
+
+    This method is used by PouchDB and other client to improve the pull
+    replications, by allowing the client to request multiple documents in one
+    request.
+
+    The usual CouchDB API for bulk gets -- a POST to _all_docs with an array of
+    docIDs in the body -- isn't suitable for the replicator because:
+
+    - there's no way to specify which specific revisions are needed (it always
+      returns the default revision),
+    - the response doesn't contain the revision history,
+    - there's no way to specify that only recently changed attachments should
+      be included (a la ``atts_since``),
+    - and attachment bodies can only be encoded inline (base64) not as MIME
+      multipart bodies.
```

Comment:
Same comment for this section as above. You're kicking sand on `_all_docs` but not describing the specific things that this endpoint does.

Try and use a neutral point of view and describe just what `_bulk_get` does first. Then, compare it with `_all_docs` and explain its unique advantages, if you like.

Reply:
OK, I've copied https://github.com/couchbase/sync_gateway/wiki/Bulk-GET#why-is-this-necessary because I didn't know what to write here. I'm going to rewrite it with your advices.

Reply:
To be honest, I don't understand why it was a new endpoint and not just some options for `POST _all_docs`. It looks like they both do the same, except one supports some options and the other some other options. I don't understand the internal of CouchDB, it probably make senses. But I have some difficulties to describe this endpoint.